components/org.wso2.carbon.identity.api.user.session/org.wso2.carbon.identity.api.user.session.v1/src/main/java/org/wso2/carbon/identity/rest/api/user/session/v1/core/SessionManagementService.java

```diff
@@ -250,4 +263,18 @@ private String getUserIdFromUser(User user) {
 
         return new UserToUniqueId().apply(SessionManagementServiceHolder.getRealmService(), user);
     }
+
+    private String getFederatedUserIdFromUser(User user) {
+
+        try {
+            return UserSessionStore.getInstance()
+                    .getUserId(user.getUserName(), IdentityTenantUtil.getTenantId(user.getTenantDomain()),
+                            FEDERATED_USER_DOMAIN);
+        } catch (UserSessionException e) {
+            log.error(String.format(
```

Comment:
can we send some error response in such cases? similar to https://github.com/wso2/identity-api-user/pull/136/files#diff-af0e18be88eef7d50d5d6bedfa8ab18c159b1799092c365eab855e22c1297009R254

Reply:
Fixed 04c5c3c4a56236168aac360389c66da28d7e292e